pmd-xml/src/main/java/net/sourceforge/pmd/cpd/XmlLanguage.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cpd;
+
+/**
```

Comment:
You can remove these javadocs - they don't add any information.

Reply:
Agreed! I was just looking at Go! :)